src/api/app/views/webui2/webui/groups/show.html.haml

```diff
@@ -0,0 +1,57 @@
+- @pagetitle = "Group #{@group}"
+
+.card
+  %div
+    %ul.nav.nav-tabs.pt-2.px-3.flex-nowrap.collapsible{ 'role': 'tablist' }
+      %li.nav-item
+        %a.active.nav-link.text-nowrap#reviews-in-tab{ aria: { controls: 'reviews-in', selected: 'true' },
+        data: { toggle: 'tab' }, href: '#reviews-in', role: 'tab' }
+          Incoming Reviews
+      %li.nav-item
+        %a.nav-link.text-nowrap#requests-in-tab{ aria: { controls: 'requests-in' }, data: { toggle: 'tab' }, href: '#requests-in', role: 'tab' }
+          Incoming Requests
+      %li.nav-item
+        %a.nav-link.text-nowrap#all-requests-tab{ aria: { controls: 'all-requests' }, data: { toggle: 'tab' }, href: '#all-requests', role: 'tab' }
+          All Requests
+      %li.nav-item
+        %a.nav-link.text-nowrap#group-members-tab{ aria: { controls: 'group-members' }, data: { toggle: 'tab' }, href: '#group-members', role: 'tab' }
+          Group Members
+      %li.nav-item.dropdown
+        %a.nav-link.dropdown-toggle{ href: '#', 'data-toggle': 'dropdown', 'role': 'button', 'aria-expanded': 'false', 'aria-haspopup': 'true' }
+        .dropdown-menu.dropdown-menu-right
+
+  .card-body.tab-content
+    .tab-pane.show.active#reviews-in{ aria: { controls: 'reviews-in' }, role: 'tabpanel' }
+      %h3 Incoming Reviews
+      = render(partial: 'webui2/shared/requests_table', locals: { id: 'reviews_in_table', source_url: group_requests_path(@group) })
+
+    .tab-pane#requests-in{ aria: { controls: 'requests-in' }, role: 'tabpanel' }
+      %h3 Incoming Requests
+      = render(partial: 'webui2/shared/requests_table', locals: { id: 'requests_in_table', source_url: group_requests_path(@group) })
+
+    .tab-pane#all-requests{ aria: { controls: 'all-requests' }, role: 'tabpanel' }
+      %h3 All Requests
+      = render(partial: 'webui2/shared/requests_table', locals: { id: 'all_requests_table', source_url: group_requests_path(@group) })
+
+    .tab-pane#group-members{ aria: { controls: 'group-members' }, role: 'tabpanel' }
+      %h3 Group Members
+      .obs-dataTable
+        %table.table.table-striped.table-bordered.w-100#group-members-table
+          %thead
+            %tr
+              %th Login Name
+          %tbody
+            - if @group.users.any?
+              - @group.users.each do |user|
+                %tr
+                  %td= link_to(user, user_show_path(user))
+            - else
+              %tr
+                %td This group does not contain users.
+        - if User.current.is_admin? || @group.group_maintainers.where(user: User.current).exists?
+          = link_to group_edit_title_path(title: @group) do
```

Comment:
This should also be handled to complete the card, but it can be in another PR.

Reply:
I created a card so we don't forget it: https://trello.com/c/MW1IGcM7/987-edit-group-members